snap/snapcraft.yaml

```diff
@@ -51,14 +51,11 @@ parts:
         - libssl-dev
         - libpq5
         - postgresql-server-dev-all
-    configflags:
+    cmake-parameters:
         - -DFLB_DEBUG=On
         - -DFLB_OUT_KAFKA=On
```

Comment:
Do we need some of the additional dependencies for librdkafka to enable extra options?